### PR TITLE
Pm 2023 5 deps bootstrap

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

File: bplan/assets/scss/core/_container.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.container-fluid {
+  padding-left: 0;
+  padding-right: 0;
+}
```
Comment (goapunk) [inline bplan/assets/scss/core/_container.scss:4]: missing newline here

File: bplan/assets/html/app/shared/directives/nav.html [hunk 2]
```diff
@@ -14,22 +14,22 @@
...
             <div class="form-group">
                 <div class="input-group">
                     <input type="text" class="form-control" data-ng-model="searchstring" data-ng-change="removeTagOnBackspace()" placeholder="Adresse oder Bebauungsplan-Nr.">
-                    <div
-                        class="input-group-addon search-button"
+                    <button
+                        class="search-button"
                         data-ng-click="getDataForSearch()"
                         data-ng-class="{middle: searchstring}">
                         <i class="fas fa-search" aria-label="Adresse oder Bebauungsplan-Nr. suchen"></i>
```
Comment (goapunk) [inline bplan/assets/html/app/shared/directives/nav.html:25]: not sure if we need to fix it now but the search button has no visual focus

Comment (goapunk) [reply bplan/assets/html/app/shared/directives/nav.html:25]: same for the search delete button below

File: bplan/assets/html/app/shared/directives/nav.html [hunk 3]
```diff
@@ -14,22 +14,22 @@
...
-                    </div>
-                    <div
+                    </button>
+                    <button
```
Comment (goapunk) [inline bplan/assets/html/app/shared/directives/nav.html:27]: this and also probably the button above need to be div for some unknown reason. If you use button the "No entries found" text won't disappear when clicking the button. No idea why

Comment (goapunk) [reply bplan/assets/html/app/shared/directives/nav.html:27]: another issue: the `No entries found` text will move the search bar up a little

File: bplan/assets/html/app/components/map/popup.html [hunk 4]
```diff
@@ -43,16 +43,14 @@
...
-            <div data-ng-if="bplan.ausleg_www" class="col-xs-3 col-sm-3 col-md-3">
-                <div><a class="custom-btn btn-primary" target="_blank" href="{{ bplan.ausleg_www }}">Website</a></div>
-            </div>
-            <div data-ng-if="bplan.scan_www" class="col-xs-3 col-sm-3 col-md-3">
-                <div><a class="custom-btn btn-primary" target="_blank" href="{{ bplan.scan_www }}">zum B-Plan</a></div>
-            </div>
-            <div data-ng-if="bplan.grund_www" class="col-xs-3 col-sm-3 col-md-3">
-                <div><a class="custom-btn btn-primary" target="_blank" href="{{ bplan.grund_www }}">Begründung</a></div>
-            </div>
+        <div data-ng-if="bplan.ausleg_www" class="col-3">
```
Comment (goapunk) [inline bplan/assets/html/app/components/map/popup.html:46]: I think we are missing a margin 20px for top and bottom here (came with the popu-entry before)

Also the `col-3` causes the "zum B-Plan" button to wrap the text too quickly, we might need something bigger

File: bplan/templates/base.html [hunk 5]
```diff
@@ -26,20 +26,20 @@
         {% verbatim %}
         <div>
             <div class="container-fluid" ng-app="app">
-                <div class="row" ng-controller="ViewController">
+                <div ng-controller="ViewController">
```
Comment (goapunk) [inline bplan/templates/base.html:29]: without the row the nav moves quite a bit to the right. maybe we can add a negative marging to the nav so it stays the same?

**Review by goapunk — CHANGES_REQUESTED** [review]
really nice! just some minor things we should probably fix

File: bplan/assets/html/app/shared/directives/nav.html (hunk 3 above)
Comment (philli-m) [reply bplan/assets/html/app/shared/directives/nav.html:27]: can't reproduce first issue but maybe that's dues to changes i've made, if it's still happening let me know which browser

File: bplan/assets/html/app/components/map/popup.html (hunk 4 above)
Comment (philli-m) [reply bplan/assets/html/app/components/map/popup.html]: couldn't replicate but changed it so it's like the list item, as it was 3-col before i imagine the issue exists on prod too?! if still a problem hit me up

File: bplan/templates/base.html (hunk 5 above)
Comment (philli-m) [reply bplan/templates/base.html:29]: row here causes inconsistencies in padding and margin for the children so have just wrapped the nav in a row div

**Review by goapunk — APPROVED** [review]
really nice!

**Pull request merged**